### PR TITLE
Group avatar on channel screen

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: examples/SampleApp/src/screens/ChannelScreen.tsx [hunk 1]
```diff
@@ -109,10 +111,19 @@ const ChannelHeader: React.FC<ChannelHeaderProps> = () => {
             }
           }}
         >
-          <Avatar
-            {...getChannelPreviewDisplayAvatar(channel, chatClient)}
-            size={40}
-          />
+          {displayAvatar.images ? (
+            <GroupAvatar
```
Comment (vishalnarkhede) [inline examples/SampleApp/src/screens/ChannelScreen.tsx:115]: @virdesai need some help here. Not working as expected, images/content overflows the container for some reason.

File: src/v2/components/ChannelPreview/hooks/useChannelPreviewDisplayName.ts [hunk 2]
```diff
@@ -92,7 +92,14 @@ export const useChannelPreviewDisplayName = <
   const channelName = channel?.data?.name;
   const maxCharacterLength = characterLength || maxCharacterLengthDefault;
 
-  const [displayName, setDisplayName] = useState('');
+  const [displayName, setDisplayName] = useState(
+    getChannelPreviewDisplayName({
```
Comment (vishalnarkhede) [inline src/v2/components/ChannelPreview/hooks/useChannelPreviewDisplayName.ts:96]: Set initial value for displayName, to avoid blank view on load.

Comment (virdesai) [reply src/v2/components/ChannelPreview/hooks/useChannelPreviewDisplayName.ts:96]: the useEffect will technically trigger this to run on the first pass through. setting this state here as well will result in this occurring twice on the initial load which isn't ideal for performance

Comment (vishalnarkhede) [reply src/v2/components/ChannelPreview/hooks/useChannelPreviewDisplayName.ts:96]: Yeah thats what I thought. Can you see anything wrong here? https://github.com/GetStream/stream-chat-react-native/blob/sample-app-cleanup/examples/SampleApp/src/screens/ChannelScreen.tsx#L87

First render happens in empty string !!

File: examples/SampleApp/src/screens/ChannelScreen.tsx (hunk 1 above)
Comment (virdesai) [reply examples/SampleApp/src/screens/ChannelScreen.tsx:117]: @vishalnarkhede I got this working and pushed it to the v2 design branch so this PR isn't needed anymore

**Pull request closed without merging**